src/ResourceManager/TrafficManager/Commands.TrafficManager2.Test/ScenarioTests/EndpointTests.cs

```diff
@@ -95,28 +95,35 @@ public void TestRemoveNonExistingEndpointFromProfile()
             TestController.NewInstance.RunPowerShellTest("Test-RemoveNonExistingEndpointFromProfile");
         }
 
-        [Fact(Skip = "TFS#5185296")]
+        [Fact]
```

Comment:
And indeed they are, no? Doesn't the attribute on the next line do it? 
[Trait(Category.AcceptanceType, Category.CheckIn)]